src/components/shared/CollapsableLink/index.tsx

```diff
@@ -68,8 +68,8 @@ const CollapsableLink = ({
       unstyled
       data-testid="collapsable-link"
     >
-      {iconPosition === 'left' ? selectedLabel : expandIcon}
-      {iconPosition === 'left' ? expandIcon : selectedLabel}
+      {iconPosition === 'right' ? selectedLabel : expandIcon}
+      {iconPosition === 'right' ? expandIcon : selectedLabel}
```

Comment:
This change affected other instances of `<CollapsableLink /> other than readonly.  Need to make sure if you change a shared component directly like this, that it doesn't affect other components.  

`<NDABanner />` uses this component and not the icon is in the wrong position

![Screenshot 2023-07-10 at 11 08 53 AM](https://github.com/CMSgov/mint-app/assets/95709965/55672a99-947f-4218-a37b-76109fcdf3ea)